### ISSUE TITLE
Rebooting it on Ubuntu led to to be stuck

### ISSUE THREAD

**Issue by wolfen351** [felixrieseberg/windows95#76]
Thanks for making this!

I tried to shutdown and reboot it - so the nostalgia can flow freely, but ended up with a black screen and unable to start it.

I eventually tried uninstalling and reinstalling the .deb with no joy. 

A search led to finding ~/.config/windows95 which i removed and reinstalled.. And voila, it started again as new! :D

This worked - but I'm thinking the "discard state" button on the UI should at least get me back to a working state?

**Comment by wolfen351** [comment]
Yeah, I can reliably do this over and over..  But i'ts ok, now that I know to delete that .config folder, all is well :)

**Comment by felixrieseberg** [comment]
`~/.config/windows95` is indeed where everything lives on Linux — good find. A lot has changed since this was filed: v5/v6 reworked restart, shutdown, and state persistence. Restart from the Start menu works now, your machine state is saved automatically when you quit, and it survives app updates. If a machine ever gets stuck, Settings → Machine State lets you reset the state or boot from scratch.

Closing this one out — if you still see trouble on the current release, please open a new issue.
